content/developer/reference/backend/orm/changelog.rst

```diff
@@ -4,6 +4,12 @@
 Changelog
 =========
 
+Odoo Online version 19.1
+========================
+
+- New API for ir.config_parameter.
```

Comment:
It would be nice to link to the corresponding documentation, in addition to the odoo/odoo PR.